atomicfu-gradle-plugin/src/main/kotlin/kotlinx/atomicfu/plugin/gradle/AtomicfuKotlinCompilerPluginInternal.kt

```diff
@@ -0,0 +1,41 @@
+package kotlinx.atomicfu.plugin.gradle
```

Comment:
Just in case, the copyright header is not missing, but was not added intentionally because now we can, right?

Reply:
Yes, that was intentional.  And we actually can, right?)

Reply:
We can, but "it would be nice to include it, but also not strictly necessary"